### PR TITLE
NEWRELIC-6962: add versioned tests to community Prisma PR

### PR DESCRIPTION
**Pull request by jordigh**
## Links

* NEWRELIC-6962

## Details
 * Added versioned tests for prisma instrumentation

### CODE REVIEW

**Review by bizob2828 — CHANGES_REQUESTED** [review]
I was comparing the pg versioned tests and we are missing tests around slow queries.  However, our instrumentation does not currently support this so I will log a different ticket and address that now.  Overall this looks fine just suggested we add more raw queries to be sure we don't regress in future

File: test/versioned/prisma/prisma.tap.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      t.end()
+    })
+  })
+
+  t.test('Raw queries should be recorded', async (t) => {
+    // We want to try the query two ways, both with and without
+    // namespacing the table, to make sure we always parse correctly
+    // and get the table.
+    const queries = [
+      prisma.$queryRaw`SELECT * FROM "public"."User"`,
```
[the remark this answers is not in the document]
Comment (jordigh) [reply test/versioned/prisma/prisma.tap.js:139]: okok will write more tests

[the remark this answers is not in the document]
Comment (jordigh) [reply test/versioned/prisma/prisma.tap.js:140]: Okay done

File: lib/instrumentation/@prisma/client.js [hunk 2]
```diff
@@ -64,7 +64,12 @@ function extractQueryArgs(args, pkgVersion) {
   let query = ''
   try {
     if (semver.gte(pkgVersion, '4.11.0')) {
-      query = args[0].args[0][0]
+      query = args[0].args[0]
```
Comment (bizob2828) [inline lib/instrumentation/@prisma/client.js:67]: glad i asked you to do this 🎉

**Review by bizob2828 — APPROVED** [review]

**Pull request merged**